test/test_mps.py

```diff
@@ -157,19 +721,26 @@ def addDecorator(op, d) -> None:
 
     for op in ops:
         key = op.name + op.variant_test_name
-        if key in XFAILLIST:
+        for xfaillist in [UNIMPLEMENTED_XFAILLIST, UNDEFINED_XFAILLIST]:
```

Comment:
Improvement to do in a follow up: ensure that the unimplemented ones actually raise a NotImplementedError

Reply:
Okay